install_config/configuring_sdn.adoc

```diff
@@ -192,26 +191,41 @@ networkConfig:
 *redhat/openshift-ovs-networkpolicy* for the *ovs-networkpolicy* plug-in
 ====
 
-[[growing-the-service-network]]
-== Growing the Service Network
+[[expanding-the-service-network]]
+== Expanding the service network
 
-If you are running low on addresses in your Service network and can expand the range so that it includes the current range at the beginning of the new range, then you can:
+If you are running low on addresses in your service network, you can expand the
+range, ensuring that the current range at the beginning of the new range:
 
-. Change the `*serviceNetworkCIDR*` parameter on all
-xref:configuring-the-pod-network-on-masters[masters] in their configuration files.
-ifdef::openshift-origin[].  You can only change the number following the / to a smaller number.
-. Delete the `*clusterNetwork*` object in OpensShift:
+. xref:configuring-the-pod-network-on-masters[Change the `*serviceNetworkCIDR*`
+parameter in the configuration files for all masters]
+(*_/etc/origin/master/master-config.yaml_* by default). Change only the number
+following the `/` to a smaller number.
+
+. Delete the `*clusterNetwork*` object:
++
 ----
 $ oc delete clusternetwork default
 ----
-. Restart the *origin-master-api* and *origin-master-controller*  services on masters .
+
+. Restart the master host services:
++
+ifdef::openshift-origin[]
+----
+# systemctl restart origin-master-api origin-master-controllers
+----
 endif::[]
 ifdef::openshift-enterprise[]
-. Restart the *atomic-openshift-master-api* and *atomic-openshift-master-controllers*
-on masters.
+----
+# systemctl restart atomic-openshift-master-controllers
+----
 endif::[]
-. Update the inventory file to change the openshift_portal_net to the new CIDR and run the playbook `*config.yml*` to redeploy the cluster
-. Evacuate each node, one by one, restart it, and then re-enable it
+
+. Update the value of the `openshift_portal_net` variable in the Ansible
```

Comment:
I'd make this two steps and include the command to run the Ansible playbook.

Reply:
Hmm @knobunc which playbook are we referring to? The master one? So:

----
# ansible-playbook /usr/share/ansible/openshift-ansible/playbooks/byo/openshift-master/config.yml
----

Reply:
Yes, I think that's the right one.  @sdodson is that correct?

Reply:
Yes that's correct.

Reply:
Thanks @sdodson @knobunc ! I'll update and merge.